data-api/schemas/crm14.yml

```diff
@@ -808,7 +982,7 @@ Crm15EmployerDTO:
     salaryDisplay:
       type: string
     salaryTax:
-      type: Integer
+      type: integer
```

Comment:
I think this value should be double. I don't think an integer covers all the possibilities